zebedee-cms/src/main/java/com/github/onsdigital/zebedee/filters/AuthenticationFilter.java

```diff
@@ -1,6 +1,6 @@
 package com.github.onsdigital.zebedee.filters;
 
-import com.github.davidcarboni.restolino.framework.Filter;
+import com.github.davidcarboni.restolino.framework.PreFilter;
```

Comment:
NB. Filter has been renamed PreFilter in ons fork of restolino to make a distinction with new PostFilter functionality.
https://github.com/ONSdigital/restolino/commit/f1e61a025f8b48f9fa30af33e7ed4394441ddbfa